### PR TITLE
Acceptance tests updates

### PR DESCRIPTION
**Pull request by sebworks**
Acceptance tests updates

## Changes
- Modified acceptance tests to remove controls flow. You can read more about it here: https://github.com/angular/protractor/blob/master/docs/control-flow.md

- Modified `test/browser_tests/page_objects/base-page.js` to reduce the transition duration time to .1ms. This should help with timeout issues.
- Modified `test/browser_tests/page_objects/base-page.js` to remove external scripts after a page is loaded.

## Removals
- Removed `wagtail-admin` tests because no one ever ran them and they provided little value.
 
## Testing

1. `Run  gulp test:acceptance --fast`

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Internet Explorer 8, 9, 10, and 11
- [ ] Edge
- [ ] iOS Safari
- [ ] Chrome for Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: test/browser_tests/cucumber/step_definitions/header.js [hunk 1]
```diff
@@ -55,39 +55,36 @@ Then( /the header organism (should|shouldn't) display the (.*)/,
...
 
 Then( /the mega-menu search form (shouldn't|should)/,
-
   function( displayElement ) {
-    browser.sleep( 500 );
 
     return expect( _dom.globalSearchContent.getAttribute( 'aria-expanded' ) )
       .to.eventually
-      .equal( shouldShouldnt( displayElement ).toString() );
+      .equal( shouldShouldnt( displayElement ).toString( ) );
```
Comment (anselmbradford) [inline test/browser_tests/cucumber/step_definitions/header.js]: Why these changes in this file? 
We don't use a space in an empty parameter list elsewhere.

File: test/browser_tests/cucumber/step_definitions/mega-menu.js [hunk 2]
```diff
@@ -28,129 +28,84 @@ Before( function() {
...
+  await EC.not( EC.elementToBeClickable( _dom.contentPolyCom ) );
+  await browser.sleep( 500 );
 
-  return browser.wait(
-    EC.not( EC.elementToBeClickable( _dom.contentPolyCom ) ) )
-    .then( () => {
-      expect( _dom.contentPolyCom.isDisplayed() )
-        .to.eventually
-        .equal( false );
+  // await expect( _dom.contentPolyCom.isDisplayed() )
```
[the remark this answers is not in the document]
Comment (sebworks) [reply test/browser_tests/cucumber/step_definitions/mega-menu.js]: yep, I'll add a TODO here.

File: test/browser_tests/page_objects/base-page.js [hunk 3]
```diff
@@ -1,29 +1,65 @@
...
+      const style = document.createElement( 'style' );
+      style.type = 'text/css';
+      style.innerHTML = '* { transition-duration: .1ms !important; }';
+      document.body.appendChild( style );
+    }
+
+    return browser.executeScript( _disableAnimations );
+  }
+
+  setElements() { } // eslint-disable-line no-empty-function
```
Comment (anselmbradford) [inline test/browser_tests/page_objects/base-page.js]: Let's put a code comment in the function that describes why it is empty and remove the eslint disabled rule.

File: test/browser_tests/conf.js [hunk 4]
```diff
@@ -31,7 +31,7 @@ function _chooseSuite( params ) {
      won't launch several identical browsers performing the same tests. */
   let capabilities = defaultSuites.essential;
 
-  if ( envvars.HEADLESS_CHROME_BINARY ) {
+  if ( _useHeadlessChrome() ) {
```
Comment (anselmbradford) [inline test/browser_tests/conf.js]: This change is so `headless` can be passed as a command-line argument? Is that correct?

File: test/browser_tests/.eslintrc [hunk 5]
```diff
@@ -7,8 +7,11 @@ globals:
...
   element: true
 rules:
+  new-cap: 0
   no-console: 0
   no-unused-expressions: 0
   max-nested-callbacks: 0
   require-jsdoc: 0
   max-statements: 0
+parserOptions:
+  ecmaVersion: 8
```
[the remark this answers is not in the document]
Comment (sebworks) [reply test/browser_tests/.eslintrc]: I'm not sure it makes sense there because we are beholden to the browsers we support.

File: test/browser_tests/cucumber/step_definitions/application.js [hunk 6]
```diff
@@ -4,10 +4,10 @@ const basePage = new BasePage();
 const { Then, When } = require( 'cucumber' );
 const { expect } = require( 'chai' );
```
[the remark this answers is not in the document]
Comment (sebworks) [reply test/browser_tests/cucumber/step_definitions/application.js]: Not quite there yet, I'll give it a go in a separate PR.

File: test/browser_tests/conf.js (hunk 4 above)
Comment (sebworks) [reply test/browser_tests/conf.js]: Yep, that is now supported in stable Chrome.

File: test/browser_tests/cucumber/step_definitions/global-search.js [hunk 7]
```diff
@@ -82,80 +82,63 @@ Then( /it (should|shouldn't) have a clear button label/,
...
-
-    return browser.wait( expectedCondition )
-      .then( () =>
-        expect( _dom.content.isDisplayed() )
-          .to.eventually
-          .equal( shouldShouldnt( haveInput ) )
-      );
+    return expect( _dom.content.isDisplayed() )
+      .to.eventually
+      .equal( shouldShouldnt( haveInput ) );
```
[the remark this answers is not in the document]
Comment (sebworks) [reply test/browser_tests/cucumber/step_definitions/global-search.js]: fixed in #3946.

File: test/browser_tests/shared_objects/related-links-section.js [hunk 8]
```diff
@@ -4,10 +4,10 @@ const relatedLinksSection = {
 
   relatedLinksSection: _relatedLinksSection,
 
-  relatedLinksSectionTitles: _relatedLinksSection.all( by.css( 'h2' ) ),
+  relatedLinksSectionTitles:  _relatedLinksSection.all( by.css( 'h2' ) ),
```
[the remark this answers is not in the document]
Comment (sebworks) [reply test/browser_tests/shared_objects/related-links-section.js]: fixed in https://github.com/cfpb/cfgov-refresh/pull/3946/commits/0ca95b5ffee295107d8aba29967a105fe641f698.

File: test/browser_tests/page_objects/base-page.js (hunk 3 above)
Comment (sebworks) [reply test/browser_tests/page_objects/base-page.js]: fixed in https://github.com/cfpb/cfgov-refresh/pull/3946/commits/7c6049fbf5872dd105ede2ba05512b407babad55.
